libarchive/archive_write_set_format_pax.c

```diff
@@ -62,6 +62,9 @@ struct pax {
 	struct sparse_block	*sparse_tail;
 	struct archive_string_conv *sconv_utf8;
 	int			 opt_binary;
+
+	unsigned flags;
+#define PAX_FLAG_GNUTAR  1
```

Comment:
There should be two bits here:
  * WRITE_SCHILY_XATTR
  * WRITE_LIBARCHIVE_XATTR